assets/src/edit-story/components/form/color/color.js

```diff
@@ -70,9 +70,10 @@ function ColorInput({
         colorPickerActions={colorPickerActions}
         changedStyle={changedStyle}
       />
-      {hasOpacity && (
+      {/* {hasOpacity && (
         <OpacityPreview value={value} onChange={handleOpacityChange} />
-      )}
+      )} */}
```

Comment:
Debugging leftover? Should remove stuff instead of commenting things out

Reply:
Good catch, bring back!